tools/cibadmin.c

```diff
@@ -759,9 +758,7 @@ main(int argc, char **argv)
             goto done;
         }
 
-        version = crm_element_value(input, PCMK_XA_CRM_FEATURE_SET);
-        digest = pcmk__digest_xml(input, true, version);
-        fprintf(stderr, "Versioned (%s) digest: ", version);
+        digest = pcmk__digest_xml(input, true);
```

Comment:
Put dropping the fprintf() in a separate commit so we can call out that the behavior is changing

Reply:
Will do. Let me know if we also want intermediate commits for `cibadmin` and `crm_diff` to say they now unconditionally output v2 digests... but digest format was never advertised AFAIK.

Reply:
Nah that's low level enough to ignore. It's still versioned, there's just one version ...